augly/image/functional.py

```diff
@@ -1759,6 +1759,58 @@ def shuffle_pixels(
     return imutils.ret_and_save_image(aug_image, output_path)
 
 
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    level: float = 0.5,
+    axis: str = "x-axis",
+    metadata: Optional[List[Dict[str, Any]]] = None,
+) -> Image.Image:
+
+    """
+    Skew image with respect to X-axis or Y-axis
+
+    @param image: the path to an image or a variable of type PIL.Image.Image
+        to be augmented
+
+    @param output_path: the path in which the resulting image will be stored.
+        If None, the resulting PIL Image will still be returned
+
+    @param level: Level of the operation that will be in [0, The maximum value of level]
+
+    @param axis: Axis along which the image will be skewed; can be 'x-axis' or 'y-axis'
+
+    @param metadata: if set to be a list, metadata about the function execution
+        including its name, the source & dest width, height, etc. will be appended
+        to the inputted list. If set to None, no metadata will be appended or returned
+
+    @returns: the augmented PIL Image
+    """
+
+    image = imutils.validate_and_load_image(image)
+    func_kwargs = imutils.get_func_kwargs(metadata, locals())
+
+    w, h = image.size
+
+    if np.random.uniform() > 0.5:
+        level = -level
+
```

Comment:
You need to not call `np.random` at all in this function in order for the behavior to be reproducible. The user can do their own random sampling to choose a value of `level`, either positive or negative, before calling the augmentation.

```suggestion
```